code/modules/hydroponics/unique_plant_genes.dm

```diff
@@ -439,14 +439,24 @@
 		spawned_simplemob.melee_damage_lower += round(our_seed.potency * mob_melee_multiplier)
 		spawned_simplemob.melee_damage_upper += round(our_seed.potency * mob_melee_multiplier)
 		spawned_simplemob.move_to_delay -= round(our_seed.production * mob_speed_multiplier)
+
+	if(isbasicmob(spawned_mob))
+		var/mob/living/basic/spawned_basicmob = spawned_mob
+		if(spawned_basicmob.basic_mob_flags & HOSTILE_MOB)
```

Comment:
Don't bother with this, the `ishostile` check is just old simplemob crust. Just set the vars regardless

Having a flag or something to check "this is a hostile mob" is a good idea but I'd rather it not be half baked (set on only one mob arbitrarily)